platform/ui/src/contextProviders/DialogProvider.js

```diff
@@ -114,62 +125,84 @@ const DialogProvider = ({ children, service }) => {
     });
   };
 
+  const validCallback = callback => callback && typeof callback === 'function';
+
   return (
-    <DialogContext.Provider value={{ create, dismiss, dismissAll, dialogs }}>
+    <DialogContext.Provider value={{ create, dismiss, dismissAll, noDialogs }}>
       <div className="DraggableArea">
         {dialogs.map(dialog => {
           const {
-            id,
-            content: Dialog,
-            position /* Position of the dialog. {{x: 0, y: 0}} */,
-            defaultPosition,
+            id = null,
```

Comment:
Is `id` even allowed to be empty? Why default to `null`?

Reply:
Its not, updated, thanks.